Header.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { View, Text, StyleSheet, Button } from "react-native";
+
+export const Headers = () => (
```

Comment:
Please try to be consistent between the name of the **file** and the name of the **component**. In this case, both component and file should be named `Header`. 

Don't you think that `Headers` is a plural name? You are indicating that there are many, but it is actually only one. When creating components, generally we use singular names since we are abstracting a set of components to a new Component that we can re-use all around our code 💪